lib/chef/file_access_control/unix.rb

```diff
@@ -280,6 +282,9 @@ def uid_from_resource(resource)
         return nil
       end
 
+      def suid_bit_set
+          return mode_to_s(target_mode)[-4] != '0'
```

Comment:
this means the setuid, setgid, or the sticky-bit (or a combination of them) is set... if you are *only* concerned about the setuid bit you should check for explicitly (also without casting to a string)

``` ruby
def suid_bit_set
  target_mode & 04000 > 0
end
```